spec/git_reflow/all-approval/git_reflow_spec.rb

```diff
@@ -18,6 +18,11 @@
   let(:existing_pull_request)  { GitReflow::GitServer::GitHub::PullRequest.new existing_pull_requests.first }
 
   before do
+
```

Comment:
Did you intend to move this file?  There's quite a lot of duplicated tests here (e.g. `.deploy`).  Let's use the default minimum approval setting and keep this named as is.